transformers/shared/constants/signature_test.go

```diff
@@ -58,14 +58,14 @@ var _ = Describe("Signature constants", func() {
 		Expect(FlipKickSignature()).To(Equal("0xc84ce3a1172f0dec3173f04caaa6005151a4bfe40d4c9f3ea28dba5f719b2a7a"))
 	})
 
-	It("generates tick signature", func() {
-		Expect(TickSignature()).To(Equal("0xfc7b6aee00000000000000000000000000000000000000000000000000000000"))
-	})
-
 	It("generates flop kick signature", func() {
 		Expect(FlopKickSignature()).To(Equal("0x7e8881001566f9f89aedb9c5dc3d856a2b81e5235a8196413ed484be91cc0df6"))
 	})
 
+	It("generates tick signature", func() {
```

Comment:
If we're moving this, I'd be tempted to move it further down so that the ordering is alphabetical by event